.buildkite/pipeline.yml

```diff
@@ -1,3 +1,28 @@
+env:
+  GITHUB_TOKEN: ${VAULT_GITHUB_TOKEN}
+
 steps:
-  - label: "E2E Test"
-    command: echo "TODO"
+  - label: ":buildkite:	Upload E2E pipeline"
+    if: build.branch == 'main' || build.source == "ui"
+    env:
+      OBLT_CLI_VERSION: 7.3.0
+    command: |
+      buildkite-agent pipeline upload <<-YAML
```

Comment:
What's the reason for using the dynamic approach?

Reply:
aha... I said nothing

Reply:
I didn't know how to provide the current commit as plugin version in `- elastic/oblt-cli#<version>`.

Or am I overthinking this?

Reply:
wait... cannot you use it without using the dynamic approach?

Reply:
Yup. I was overthinking it.

It works.. https://github.com/elastic/oblt-cli-buildkite-plugin/pull/2/commits/8bc9ed553240cee70a7506746c0724bfea1814c7

Reply:
Thank you for the hint!